cmd/eval.go

```diff
@@ -467,6 +469,11 @@ func evalOnce(ctx context.Context, ectx *evalContext) pr.Output {
 		result.Coverage = &report
 	}
 
+	if ectx.params.strict {
+		err := enabledStrictFlag(ectx.params)
```

Comment:
Interesting approach... I'd have expected that we would use the `rego.Strict(y bool)` you've added to pass that along to the compiler used within the `rego` package. I think I'd prefer that over having a compiler instantiated here. Has that approach brought up any problems?

Reply:
> Interesting approach... I'd have expected that we would use the `rego.Strict(y bool)` you've added to pass that along to the compiler used within the `rego` package. I think I'd prefer that over having a compiler instantiated here. Has that approach brought up any problems?

Hmm.. Good point :+1: let me try that and come back.

Reply:
So, to somewhat of my dismay I think I overthought this and neglected a much tidier (and easier) route as you mentioned which can be seen in my new PR.

The issue with doing:

```
	if params.strict {
		regoArgs = append(regoArgs, rego.Strict(params.strict))
	}
```

Was that it kept applying strict to the query as well, but I didn't think about just switching this off in the query compiler :upside_down_face: (thanks)